### DISCUSSION TITLE
about window shadow effects

### DISCUSSION THREAD

**Discussion by nomi2015** [AvaloniaUI/Avalonia#12296] [Q&A]
Custom windows don't have a shadow effect, but lower versions have it.

**Comment by yll690** [comment]
There may be changes about `ExtendClientAreaToDecorationsHint`, `ExtendClientAreaChromeHints` and `SystemDecorations` properties. Try to change them. My window has these values: `True`, `SystemChrome`, `BorderOnly` and it has a shadow effects on Windows.
